bin/dlang/codecov

```diff
@@ -4,11 +4,14 @@
 # (See accompanying file LICENSE.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 #
-# Build D applications
+# Report code coverage to codecov for D projects, it is a wrapper on the beaver
+# codecov script but adds more default options on top of it and collects
+# reports automatically (so you don't have to pass BEAVER_CODECOV_REPORTS).
 set -eu
 
 # First check if there are any reports at all
-if test "$(echo .*.lst)" = ".*.lst"
+export BEAVER_CODECOV_REPORTS=".*.lst"
```

Comment:
DMD generated lst files don't necessarily start with `.` - it only happens for files that are not compiled from current working dir as a root import path AFAIK. So, for example, integration test module reports will not have it.

Reply:
Mmm, OK, but then this is a pre-existing bug :)

Reply:
BTW, can you open an issue about this? It should be fixed in v0.3.x and this targets v0.4.x.